### ISSUE TITLE
Use new Github features

### ISSUE THREAD

**Issue by plato2000** [DRMF/DRMF-Seeding-Project#88]
Recently, Github has added the review pull request feature (which I am using already) and the Projects feature. I think we should use both of these features to increase productivity.  The pull request review allows people who are reviewing the code to ask for changes before approval, which acts like any other check before merging (like Travis or Landscape). This makes it easier for multiple people to ask for different changes and to see them all bundled together with comments. This is easy to use and there is no reason not to.

The other change is the Github project board. Here, you can create different "projects" and assign pull requests, issues, and notes to each of them. This will allow for easier communication about certain projects without things getting lost in the pull request comments. Also, it will allow for better organization of what comments and pull requests belong to who and what project. I think implementing this would improve both productivity and ease of management.

Thoughts? @HowardCohl @physikerwelt

**Comment by physikerwelt** [comment]
Yes I'm all for it. I have recently tested it to manage the issues for my phdthesis https://github.com/TU-Berlin/schubotzPhd/projects/1
